.prettierignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
added this to ignore linting the manifest files for now -- is there a better way to add custom linting rules (or ways to ignore certain lines of code in the CI checks)? the tests complain because of the triple braces used for mustache